setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import setup, find_packages
+import validator
+
+
+setup(
+    name='content-validator',
+    version=validator.__version__,
+    description='Validate XML against the Libero content model',
+    packages=find_packages(exclude=['tests']),
+    include_package_data=True,
+    install_requires=['lxml'],
+    license='MIT',
+    url='https://github.com/libero/content-validator.git',
+    maintainer='eLife Sciences Publications Ltd.',
+    maintainer_email='tech-team@elifesciences.org',
```

Comment:
We have a specific email address `libero-admin@elifesciences.org` now that could be used